h/schemas/annotation.py

```diff
@@ -391,6 +391,14 @@ class SearchParamsSchema(colander.Schema):
                        a URN representing another kind of resource such as DOI
                        (Digital Object Identifier) or a PDF fingerprint.""",
     )
+    uri_parts = colander.SchemaNode(
+        colander.Sequence(),
+        colander.SchemaNode(colander.String()),
+        name='uri.parts',
```

Comment:
The `name` is normally inferred from the attribute name but it has to be added explicitly here since periods are not allowed in statically declared attribute names.